mobify/platform-scaffold
# WEB-1534: PLP microdata

This PR adds microdata for the product schema to the PLP. It also removes any remaining instances of the product schema containing prices that include currency symbols (the product microdata required prices to not include currency symbols). 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1534
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/857

## Changes
- Updated merlins and SFCC product parsers to not include currency symbols
- Updated components that display product prices to pull currency from the store separate from prices
- Updated product list contents to include microdata

## Todos

## How to test-drive this PR
- cd into the SDK branch WEB-1534__component-support-structured-data
- run `npm run prod:build`
- run `npm link`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm start`
- Preview merlins potions
- navigate to a PLP
- copy the markup for the PLP (starting at the '.t-product-list__container' container)
- paste the markup into the structured data testing tool: https://search.google.com/structured-data/testing-tool
- the data should validate and contain all the necessary properties
- repeat the steps using the SFCC connector and SFCC demo site
